src/android/build.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
+    implementation files('src/main/libs/android-biometric.aar')
```

Comment:
Sorry again but this is file built from the library I guess? We should make an annotation and mention that this is under Apache License.

Reply:
I've made some comments under this line. Thanks.

Reply:
Perfect thanks